<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>App</class>
 <widget class="QMainWindow" name="App">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>BeatVault</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="pathButton">
        <property name="text">
         <string>Select Beat Saber Path</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="coverLabel">
        <property name="minimumSize">
         <size>
          <width>480</width>
          <height>480</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>480</width>
          <height>480</height>
         </size>
        </property>
        <property name="text">
         <string>(Image goes here)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTextEdit" name="textEdit"/>
      </item>
      <item>
       <widget class="QListWidget" name="songList"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuBeatVault_0_0_1">
    <property name="title">
     <string>BeatVault</string>
    </property>
    <widget class="QMenu" name="menuAbout">
     <property name="title">
      <string>About</string>
     </property>
     <addaction name="actionBeatVault_v0_0_1"/>
     <addaction name="actionCopyright_Parker_Van_Roy_2021"/>
    </widget>
    <addaction name="menuAbout"/>
   </widget>
   <addaction name="menuBeatVault_0_0_1"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionBeatVault_v0_0_1">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>BeatVault v0.0.1</string>
   </property>
  </action>
  <action name="actionCopyright_Parker_Van_Roy_2021">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Copyright Parker Van Roy 2021</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
